Add example Docker file and wrapper script (#131)
graphile/migrate: docs/docker/README.md

# Running Graphile Migrate from a container

> When working in a team it can be useful to package `graphile-migrate` and the
> postgres tooling into a docker file so that everyone has easy access to the
> same versions. This is also helpful if you're using Migrate as part of a larger
> non-nodejs based project.

Reviewer: ```suggestion
When working in a team it can be useful to package `graphile-migrate` and the
Postgres tooling into a Docker file so that everyone has easy access to the
same versions. This is also helpful if you're using Migrate as part of a larger
non-Node based project.
```

> For these purposes we provide an [example docker file](./Dockerfile) in the
> `docs/docker` directory in the source tree.  This uses the latest released
> version of graphile-migrate from `npm` and packages it together with the
> necessary nodejs and postgres tools. You can build the docker file from the
> root of the repository using a command like

Reviewer: ```suggestion
For these purposes we provide an [example Dockerfile](./Dockerfile) in the
`docs/docker` directory in the source tree. This uses the latest released
version of `graphile-migrate` from `npm` and packages it together with the
necessary Node and Postgres tools. You can build the Dockerfile from the
root of the repository using a command like such as:
```
